src/solution.jl

```diff
@@ -0,0 +1,33 @@
+# π, σ and ρ do not really make sense alone so only
+# their product will T, h, d, e is stored
+mutable struct Solution
```

Comment:
Can you expand a bit on these definitions? Provide a doc-string?

Reply:
I have hidden this into the `nlds.jl` file and replaced it with an `AbstractSolution` interface